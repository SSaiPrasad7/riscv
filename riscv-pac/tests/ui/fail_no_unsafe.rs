#[riscv_pac::pac_enum(InterruptNumber)]
#[derive(Clone, Copy, Debug, PartialEq)]
enum Interrupt {
    I1 = 1,
    I2 = 2,
    I4 = 4,
}

fn main() {}
