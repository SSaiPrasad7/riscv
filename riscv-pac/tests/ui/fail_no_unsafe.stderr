error: expected `unsafe`
 --> tests/ui/fail_no_unsafe.rs:1:23
  |
1 | #[riscv_pac::pac_enum(InterruptNumber)]
  |                       ^^^^^^^^^^^^^^^
